### PR TITLE
Allow reply in quote block

### LINKED ISSUE #5283: Cannot add newline into quoted reply
### Expected behavior
I am able to add a newline into a quoted reply. So that I can reply with context.

Screenshot 2021-07-05 at 11 09 16

### Actual behavior

I can't insert a newline and get a JS error. Unclear if related:

```
vue.runtime.esm.js:1897 TypeError: Cannot read property 'includes' of null
    at c (text.js:73)
    at a.onInputChanged (Composer.vue:673)
    at Wt (vue.runtime.esm.js:1863)
    at a.n (vue.runtime.esm.js:2184)
    at Wt (vue.runtime.esm.js:1863)
    at a.t.$emit (vue.runtime.esm.js:3903)
    at a.onInput (TextEditor.vue:198)
    at Wt (vue.runtime.esm.js:1863)
    at a.n (vue.runtime.esm.js:2184)
    at Wt (vue.runtime.esm.js:1863)
```

### Mail app

c.nc.c. I tried this on latest Chrome and Safari on macOS.

### PR DESCRIPTION
**Pull request by kesselb**
Fix #5283 

The relevant change is that `quota` is not longer defined as limit but block element. https://ckeditor.com/docs/ckeditor5/latest/framework/guides/deep-dive/schema.html#limit-elements as explanation what a limit element in CKEditor is. 

**Test plan:**
- Pick a message
- Press reply
- Try to write something in the quoted message

### CODE REVIEW

File: src/ckeditor/quote/QuotePlugin.js [hunk 1]
```diff
@@ -32,9 +32,7 @@ export default class Quote extends Plugin {
 		const schema = this.editor.model.schema
 
 		schema.register('quote', {
-			allowIn: '$root',
```
Comment (ChristophWurst) [inline src/ckeditor/quote/QuotePlugin.js:35]: could you elaborate on this change? :)

Comment (kesselb) [reply src/ckeditor/quote/QuotePlugin.js:35]: I did in the description ;)

**Review by ChristophWurst — APPROVED** [review]

File: src/ckeditor/quote/QuotePlugin.js (hunk 1 above)
Comment (ChristophWurst) [reply src/ckeditor/quote/QuotePlugin.js:35]: not when I first reviewed :wink:

Comment (kesselb) [reply src/ckeditor/quote/QuotePlugin.js:35]: You are just to fast :racing_car:

**Pull request merged**